src/xrpl-account.js

```diff
@@ -123,6 +123,33 @@ class XrplAccount {
         return this.#submitAndVerifyTransaction(tx, options);
     }
 
+    setAccountFields(fields, options = {}) {
```

Comment:
Rather than keeping name, value pairs for 'fields', it's cleaner to just provide a json object.
```
{ "Domain", "www.mydomain.com", "Field2", "value2", etc...)
```

1. If some fields need to be hex encoded, this function should do it inside, so the caller can pass the simple plain text string.
2. The "Flags" can be handled in a convenient way. We could get the "Flags" object within our "fields" object as a set of true/false values and then translate those flags to "SetFlag" and "ClearFlag" fields of the xrpl library.

Example:
```
'fields' parameter = {
  Domain: "www.mydomain.com",
  Flags: { asfDefaultRipple: false, asfAuthorizedMinter: true, asfDisableMaster: true }
};

resulting xrpl tx fields = {
  Domain: "<hex encoded domain>",
  SetFlags: xrpl.AccountSetAsfFlags["asfDisableMaster"],
  ClearFlags: xrpl.AccountSetAsfFlags["asfAuthorizedMinter"] | xrpl.AccountSetAsfFlags["asfDefaultRipple"],
};
```
2. This function is a better alternative to specific functions 'setDefaultRippling()' and 'setMessageKey()'. So you can remove them and change the callers of those functions to use 'setAccountFields()' appropriately.